qcodes/instrument_drivers/yokogawa/GS200.py

```diff
@@ -161,6 +161,62 @@ def update_measurement_enabled(self, unit: str,
             self.measure.unit = 'V'
 
 
+class GS200Program(InstrumentChannel):
+    """
+    """
+    def __init__(self, parent: 'GS200', name: str) -> None:
+        super().__init__(parent, name)
+        self._repeat = 1
+        self._file_name = None
+
+        self.add_parameter("interval",
+                           label="the program interval time",
+                           get_cmd=":PROG:INT?",
+                           set_cmd=":PROG:INT {}")
+
+        self.add_parameter("slope",
+                           label="the program slope time",
+                           get_cmd=":PROG:SLOP?",
+                           set_cmd=":PROG:SLOP {}")
+
+        self.add_parameter("trigger",
+                           label="the program trigger",
+                           get_cmd=":PROG:TRIG?",
+                           set_cmd=":PROG:TRIG {}",
+                           vals=Enum('normal', 'mend'))
+
+        self.add_parameter("save",
+                           label="save the program",
+                           set_cmd=":PROG:SAVE {}")
+
+        self.add_parameter("load",
+                           label="load the program",
+                           get_cmd=":PROG:LOAD?",
+                           set_cmd=":PROG:LOAD {}")
```

Comment:
Could you add some explanation of how load and save works ? it is not clear from the docscrings and the examples. Can we validate the input for save and load/